src/components/track-tile/TrackTileMetadata.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react'
+
+import { ID } from 'audius-client/src/common/models/Identifiers'
+import { CoverArtSizes } from 'audius-client/src/common/models/ImageSizes'
+import { Remix } from 'audius-client/src/common/models/Track'
+import { User } from 'audius-client/src/common/models/User'
+import {
+  Animated,
+  GestureResponderEvent,
+  Pressable,
+  StyleSheet,
+  View
+} from 'react-native'
+
+import IconVolume from 'app/assets/images/iconVolume.svg'
+import Skeleton from 'app/components/skeleton'
+import Text, { AnimatedText } from 'app/components/text'
+import UserBadges from 'app/components/user-badges/UserBadges'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { flexCol, flexRow, flexRowCentered } from 'app/styles'
+import { ThemeColors, useThemeColors } from 'app/utils/theme'
+
+import TrackTileArt from './TrackTileArt'
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    metadata: {
+      ...flexRow()
```

Comment:
these are fun! do they need to be functions even?

Reply:
They don't need to be functions, some of them might take arguments so I figured it would make them easier to use if they were all consistent. `fontWeight` is one example

Reply:
ahh perfect, thanks